runhouse/servers/env_servlet.py

```diff
@@ -227,61 +261,66 @@ def _get_env_cpu_usage(self, cluster_config: dict = None):
         except psutil.NoSuchProcess:
             env_memory_usage = {}
 
-        return (env_memory_usage, node_name, total_memory, env_servlet_pid, node_ip)
+        return (env_memory_usage, node_name, total_memory, self.pid, node_ip)
```

Comment:
```suggestion
        return env_memory_usage, node_name, total_memory, self.pid, node_ip
```

also why do we need to return `self.pid` here?

Reply:
prob will be useful for multi-node clusters, will add it in the relevant PR.